go/private/context.bzl

```diff
@@ -391,6 +391,7 @@ def go_context(ctx, attr = None):
     stdlib = None
     coverdata = None
     nogo = None
+    gc_goopts = None
```

Comment:
Now that gc_goopts is read from `ctx.attr.gc_goopts[BuildSettingInfo].value`, You don't need this anymore, right?

Reply:
That's right, done!